FirebaseVertexAI/Sample/ChatSample/Screens/ConversationScreen.swift

```diff
@@ -68,9 +68,11 @@ struct ConversationScreen: View {
       InputField("Message...", text: $userPrompt) {
         Image(systemName: viewModel.busy ? "stop.circle.fill" : "arrow.up.circle.fill")
           .font(.title)
+          .padding(.bottom, 3)
       }
       .focused($focusedField, equals: .message)
       .onSubmit { sendOrStop() }
+      .padding(.bottom, 5)
```

Comment:
Instead of padding here, we should fix the padding on the InputField.

Reply:
Removed 🙂